src/plptool/gui/ProjectDriver.java

```diff
@@ -299,8 +297,11 @@ else if (X + W <= screenResolution.width && Y + H <= screenResolution.height)
 		// check for JRE version
 		
 		String tokens[] = System.getProperty("java.version").split("\\.");
-		int major = Integer.parseInt(tokens[0]);
-		int minor = Integer.parseInt(tokens[1]);
+			int major = Integer.parseInt(tokens[0]);
+			int minor = 0; // Default value to prevent index OOB exception.
```

Comment:
Should it keep the same indentation with the original file?

Reply:
```suggestion
		int major = Integer.parseInt(tokens[0]);
		int minor = 0; // Default value for JVMs without minor versions.
```